js/renovation/ui/toolbar/toolbar_props.tsx

```diff
@@ -114,6 +114,8 @@ export class ToolbarTextBoxProps {
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   @Event() onValueChanged?: EventCallback<any>;
 
+  @OneWay()
```

Comment:
Try to extract rtlEnabled in to the base ToolbarItemProps (add it if you doesn't have it) and reuse it in other item class using inheritance
@ComponentBindings()
export class ToolbarItemProps {
rtlEnabled?: boolean | undefined;
}
@ComponentBindings()
export class ToolbarCheckBoxProps inherits ToolbarItemProps{
}

Reply:
Yep, great idea. Fixed